data/settings/1.39.x/kubernetes.toml

```diff
@@ -0,0 +1,653 @@
+[[docs.tag.labels-and-taints]]
```

Comment:
Isn't this missing the new 3 settings that are supported or will they be added in a separate PR?

>Add new Kubernetes settings (https://github.com/bottlerocket-os/bottlerocket/pull/4506)

    containerLogMaxWorkers
    containerLogMonitorInterval
    singleProcessOOMKill